lighty-modules/lighty-gnmi/lighty-gnmi-sb/src/main/java/io/lighty/gnmi/southbound/mountpoint/transactions/ReadOnlyTx.java

```diff
@@ -33,7 +33,7 @@ public ReadOnlyTx(final GnmiGet getProvider) {
 
     @Override
     public void close() {
-
+        LOG.info("Method not implemented!");
```

Comment:
this is not exactly correct, no implementation is needed
change to `debug` level and extend message

Reply:
done